cmd/server.go

```diff
@@ -218,6 +210,14 @@ Config file values are overridden by environment variables which in turn are ove
 			return s.run()
 		}),
 	}
+
+	// Configure viper to accept env vars prefixed with ATLANTIS_ that can be
+	// used instead of flags.
+	s.Viper.SetEnvPrefix("ATLANTIS")
+	s.Viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
+	s.Viper.AutomaticEnv()
+	s.Viper.SetTypeByDefaultValue(true)
```

Comment:
Benefits of testing! Without this you can't set a boolean flag as an env var, ex. `ATLANTIS_ALLOW_FORK_PRS=true` won't work without this.